README.md

```diff
@@ -147,13 +137,23 @@ The breakpoints are configured in `./src/scss/settings/_settings.breakpoints.scs
 ## File Generators
 To quickly add new parts to your app like components modules etc. the syntax to you can use is
 ```sh
-yo emakinacee-react:gen type name module
+react g <scaffold> <name> [module]
```

Comment:
f5d46015b205796aca858c9d74d59861c8a52e9c - [react-cli] add 'generate' command and handle 'g' as shorthand